tests/integ/test_sagemaker_steps.py

```diff
@@ -29,7 +29,7 @@
 from sagemaker.tuner import HyperparameterTuner
 from sagemaker.processing import ProcessingInput, ProcessingOutput
 
-from stepfunctions.inputs import ExecutionInput
+from stepfunctions.inputs import ExecutionInput, StepInput
```

Comment:
```suggestion
from stepfunctions.inputs import ExecutionInput
```

Reply:
Will remove both ExecutionInput and StepInput in next commit (removed the integ test with placeholders so ExecutionInput isn't used anymore)